Improve build instructions

jarl-dk opened the pull request.

lukasmatena commented:
This does not look right. If you actually "built it from the documentation", you would not need most of the packages that you added, because the libraries would be built during the second step (Building dependencies). This is the case for zlib, glew, curl, ssl, cereal, nlopt and cgal. On the other hand, if you built against system libraries, there are quite a few still missing (wxWidgets, tbb and boost for example), but this variant is unsupported and should not be described in the guide at all.

I admit, however, that dbus and autoconf should be mentioned (m4 is part of autoconf, so it does not have to be). As for the GTK2/3 issue you are totally right, and I have already fixed this on Monday in 5c55bee (by unifying everything to the GTK3 variant).

jarl-dk commented:
I agree. I do only try to build according to the build instructions. I didn't notice that you recently changed the docs, thanks. The reason that I mis-included many dev libraries was because I try to build prior to successfully building deps in an attempt to speed up the package identification. I have now updated the [buildbox](https://github.com/jarl-dk/PrusaSlicerBuildBox) as it proves the steps found in the documentation actually works.
I have updated this PR.

lukasmatena commented:
Thanks. Since you apparently like to be  precise, I noticed that the package list is now missing `git` and there is a double "and" in the first sentence. If you update it, I will do the merge (if you don't, I will do the change myself later).

jarl-dk commented:
Thanks. Here you go...

lukasmatena commented:
Thanks.
